DESCRIPTION

```diff
@@ -36,6 +36,10 @@ Authors@R: c(
 	   email = "philippine.louail@eurac.edu",
 	   role = "aut",
 	   comment = c(ORCID = "0009-0007-5429-6846"))
```

Comment:
@pablovgd , please add a `,` here at the end of the line.

Reply:
My bad, done!